torch/csrc/jit/init.cpp

```diff
@@ -180,9 +181,12 @@ void initJITBindings(PyObject* module) {
           py::arg("module"),
           py::arg("method_name"),
           py::arg("inplace") = false)
+      .def(
+          "_jit_pass_insert_pack_unpack",
+          [](script::Module& module) { return InsertPackUnpack(module); })
       .def(
           "_jit_pass_quant_fusion",
-          [](std::shared_ptr<Graph>& g) { return QuantFusion(g); })
+          [](std::shared_ptr<Graph> g) { return QuantFusion(g); })
```

Comment:
What caused this change?

Reply:
I changed this for consistency with other passes, but I look again and saw that there are some APIs that passes `std::shared_ptr<Graph>` and some passes `std::shared_ptr<Graph>&`.

Reply:
> I changed this for consistency with other passes, but I look again and saw that there are some APIs that passes `std::shared_ptr<Graph>` and some passes `std::shared_ptr<Graph>&`.

Majority of the passes uses the reference option, so let's use it here as well. Technically, it also saves one refcount increment.

Reply:
yeah, we should use `&` option